model/src/main/kotlin/OrtResult.kt

```diff
@@ -76,6 +77,18 @@ data class OrtResult(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val data: CustomData = mutableMapOf()
 
+    @JsonIgnore
+    private val excludedProjects: Set<Identifier> =
+        getProjects()
+            .filter { project ->
+                val projectExclude = getExcludes().findProjectExclude(project, this)
+                val pathExcludes = getExcludes().findPathExcludes(project, this)
+                val isExcluded = projectExclude != null || pathExcludes.isNotEmpty()
```

Comment:
Don't you also need to check if `projectExclude.isWholeProjectExcluded == true`?